source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -178,7 +178,14 @@ int PublicKeyWrapper::luaGet(lua_State* state) {
   if (public_key_ == nullptr || public_key_.get() == nullptr) {
     lua_pushnil(state);
   } else {
-    lua_pushlightuserdata(state, public_key_.get());
+    auto wrapper =
+        Common::Crypto::Access::getTyped<Common::Crypto::PublicKeyObject>(*(public_key_.get()));
```

Comment:
You can probably also do `*public_key_.get()`.

Reply:
`*public_key_` is enough (otherwise clang_tidy will blame for unnecessary `.get()`